src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+}
+
+body {
+  font-family: 'Avenir Book', sans-serif;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  display: block;
+  margin-top: 20px;
```

Comment:
You have added a top margin to the 'search-bar' class. This is not a good practice as per the checklist. Instead of adding margins to the top and bottom, it is recommended to add them to either the top or the bottom to avoid potential margin collapse.